app/controllers/short_url_requests_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @existing_redirect = Redirect.where(from_path: @short_url_request.from_path).first
```

Comment:
OK.  The ticket talks about "other requests" rather than "an existing redirect".  Could you confirm whether this is a mistake in the ticket?